docs/future-work/motor-system-improvements/implement-efficient-saccades-driven-by-model-free-and-model-based-signals.md

```diff
@@ -9,3 +9,8 @@ In the model-free case, salient information available in the view-finder could d
 In the model-based case, two primary settings should be considered:
 - A single LM has determined that the agent should move to a particular location in order to test a hypothesis, and it sends a goal-state that can be satisfied with a saccade, rather than the entire agent jumping/teleporting to a new location. For example, saccading to where the handle of a mug is believed to be will refute or confirm the current hypothesis. This is the more important/immediate use case.
 - Multiple LMs are present, including a smaller subset of more peripheral LMs. If one of these peripheral LMs observes something of interest, it can direct a goal-state to the motor system to perform a saccade such that a dense sub-set of LMs are able to visualize the object. This is analogous to cortical feedback bringing the fovea to an area of interest.
+
+Such policies are particularly important in an unsupervised setting, where we will want to more efficiently explore objects in order to rapidly determine their identity, given we have no supervised signal to tell us whether this is a familiar object, or an entirely new one.
```

Comment:
```suggestion
Such policies are particularly important in an unsupervised setting, where we will want to explore objects more efficiently in order to rapidly determine their identity, given we have no supervised signal to tell us whether this is a familiar object, or an entirely new one.
```

Reply:
It's also important if we adjust the memory horizon parameters since with a shorter memory we need more efficient exploration.